Build 710 for main with status FAILURE

## :broken_heart: Tests Failed


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2Fmain/detail/main/710//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2Fmain/detail/main/710//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2Fmain/detail/main/710//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2Fmain/detail/main/710//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://beats_null.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/beats-ci/transactions/view?rangeFrom=2022-07-25T09:21:22.415Z&rangeTo=2022-07-25T09:41:22.415Z&transactionName=Beats/beats/main&transactionType=job&latencyAggregationType=avg&traceId=567a64af5deabd9109c7fb6a2cf42412&transactionId=f1ce54e5155c4b94)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2022-07-25T09:31:22.415+0000


* Duration: 179 min 53 sec




#### Test stats :test_tube:

| Test         | Results                         |
| ------------ | :-----------------------------: |
| Failed       | 1  |
| Passed       | 46942  |
| Skipped      | 5191 |
| Total        | 52134   |


</p>
</details>

<!-- TEST RESULTS IF ANY-->

  ### Test errors [![1](https://img.shields.io/badge/1%20-red)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2Fmain/detail/main/710//tests)
<details><summary>Expand to view the tests failures</summary><p>
    
    
##### `ExtendedWin / x-pack/auditbeat-windows-10-windows-10 / TestLinkedListRemoveOlderNoAllocs – github.com/elastic/beats/v7/x-pack/auditbeat/module/system/socket/helper`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 Failed 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 === RUN   TestLinkedListRemoveOlderNoAllocs
    linkedlist_test.go:111: 
        	Error Trace:	linkedlist_test.go:111
        	Error:      	Should be zero, but was 1
        	Test:       	TestLinkedListRemoveOlderNoAllocs
--- FAIL: TestLinkedListRemoveOlderNoAllocs (1.54s)
 
 ```
</p></details>
</ul>
    
</p></details>


<!-- STEPS ERRORS IF ANY -->














### Steps errors [![6](https://img.shields.io/badge/6%20-red)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2Fmain/detail/main/710//pipeline)
  <details><summary>Expand to view the steps failures</summary>
  <p>


  
  
  
  
##### `Docker login`
<ul>
<li>Took 0 min 6 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/main/runs/710/steps/2757/log/?start=0">here</a></li>
<li>Description: <code>          set +x          if command -v host 2>&1 > /dev/null; then            host docker.io 2>&1 > /dev/null          fi          if command -v dig 2>&1 > /dev/null; then            dig docker.io 2>&1 > /dev/null          fi          docker login -u "${DOCKER_USER}" -p "${DOCKER_PASSWORD}" "docker.io" 2>/dev/null          </code></l1>
</ul>
  
  
  
  
##### `x-pack/filebeat-arm-arm - mage build unitTest`
<ul>
<li>Took 9 min 37 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/main/runs/710/steps/26921/log/?start=0">here</a></li>
<li>Description: <code>mage build unitTest</code></l1>
</ul>
  
  
  
  
##### `x-pack/auditbeat-windows-10-windows-10 - mage build unitTest`
<ul>
<li>Took 3 min 23 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/main/runs/710/steps/35330/log/?start=0">here</a></li>
<li>Description: <code>mage build unitTest</code></l1>
</ul>
  
  
  
  
##### `x-pack/auditbeat-windows-10-windows-10 - mage build unitTest`
<ul>
<li>Took 1 min 34 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/main/runs/710/steps/37168/log/?start=0">here</a></li>
<li>Description: <code>mage build unitTest</code></l1>
</ul>
  
  
  
  
##### `x-pack/auditbeat-windows-10-windows-10 - mage build unitTest`
<ul>
<li>Took 1 min 32 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/main/runs/710/steps/37332/log/?start=0">here</a></li>
<li>Description: <code>mage build unitTest</code></l1>
</ul>
  
  
  
  
##### `Error signal`
<ul>
<li>Took 0 min 0 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/main/runs/710/steps/37442/log/?start=0">here</a></li>
<li>Description: <code>Error "hudson.AbortException: script returned exit code 1"</code></l1>
</ul>
  
  </p>
  </details>

@elastic/security-external-integrations I think this is one of your tests, failed twice in the past day:

```
 === RUN   TestLinkedListRemoveOlderNoAllocs
    linkedlist_test.go:111: 
        	Error Trace:	linkedlist_test.go:111
        	Error:      	Should be zero, but was 1
        	Test:       	TestLinkedListRemoveOlderNoAllocs
--- FAIL: TestLinkedListRemoveOlderNoAllocs (1.54s)
```

--- a/x-pack/auditbeat/module/system/socket/helper/linkedlist.go
+++ b/x-pack/auditbeat/module/system/socket/helper/linkedlist.go
@@ -2,6 +2,9 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+//go:build (linux && 386) || (linux && amd64)
+// +build linux,386 linux,amd64
+
 package helper
 
 import "time"

--- a/x-pack/auditbeat/module/system/socket/helper/linkedlist_test.go
+++ b/x-pack/auditbeat/module/system/socket/helper/linkedlist_test.go
@@ -2,6 +2,9 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+//go:build (linux && 386) || (linux && amd64)
+// +build linux,386 linux,amd64
+
 package helper
 
 import (

